packages/cli/scripts/build.ts

```diff
@@ -77,6 +77,7 @@ async function main() {
 
   // Band-aid to bundle stuff that `ncc` neglects to bundle
   await cpy(join(dirRoot, 'src/util/projects/VERCEL_DIR_README.txt'), distRoot);
+  await cpy(join(dirRoot, 'src/util/dev/builder-worker.js'), distRoot);
```

Comment:
Strangely, `ncc` is no longer including this file into the dist dir as a static asset. This line is missing from the build logs:

```
vercel:build:    2kB  dist/builder-worker.js
```